refactoring/initial_two_classes.py

```diff
@@ -32,17 +32,32 @@ def add_person(self, name, age, job):
 
     def number_of_connections(self, name):
         """Find the number of connections that a person in the group has"""
-        pass
+        if name in self.connections:
+            return len(self.connections[name])
+        else:
+            return 0
```

Comment:
This is certainly one way to deal with it, the other would be to throw an exception